
Sistema -> 25/11/2012 - 22:43:26
	ce.model.dao.RepositorioCategoria.excluir(): com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Cannot delete or update a parent row: a foreign key constraint fails (`estoque`.`produto`, CONSTRAINT `produto_fkCodCateg` FOREIGN KEY (`codCateg`) REFERENCES `categoria` (`CodCateg`))
joao.souza -> 25/11/2012 - 22:43:26
	ce.model.regra.ControladorCategoria.excluir(): Não é possível excluir a entrada informada, pois a mesma
 está sendo usanda por um ou mais produto!
Sistema -> 25/11/2012 - 22:46:37
	ce.model.dao.RepositorioCategoria.excluir(): com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Cannot delete or update a parent row: a foreign key constraint fails (`estoque`.`produto`, CONSTRAINT `produto_fkCodCateg` FOREIGN KEY (`codCateg`) REFERENCES `categoria` (`CodCateg`))
joao.souza -> 25/11/2012 - 22:46:37
	ce.model.regra.ControladorCategoria.excluir(): Não é possível excluir a categoria informada, pois a mesma
 está sendo usanda por um ou mais produto!
Sistema -> 25/11/2012 - 22:48:07
	ce.model.dao.RepositorioCategoria.excluir(): com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Cannot delete or update a parent row: a foreign key constraint fails (`estoque`.`produto`, CONSTRAINT `produto_fkCodCateg` FOREIGN KEY (`codCateg`) REFERENCES `categoria` (`CodCateg`))
joao.souza -> 25/11/2012 - 22:48:07
	ce.model.regra.ControladorCategoria.excluir(): Não é possível excluir a categoria informada, pois a mesma
 está sendo usanda por um ou mais produto!
root -> 27/11/2012 - 17:43:46
	ce.util.GerenciadorConexao.conectar(): java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
joao.souza -> 27/11/2012 - 17:43:46
	ce.model.regra.ControladorUsuario.validarLogin(): No momento não é possível validar o login.
adm.sys -> 02/12/2012 - 20:40:02
	ce.model.regra.ControladorUsuario.validarLogin(): Senha inválida!
adm.sys -> 02/12/2012 - 20:40:11
	ce.model.regra.ControladorUsuario.validarLogin(): Senha inválida!
root -> 03/12/2012 - 11:57:15
	ce.util.GerenciadorConexao.conectar(): java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
joao.souza -> 03/12/2012 - 11:57:15
	ce.model.regra.ControladorUsuario.validarLogin(): No momento não é possível validar o login.
Sistema -> 03/12/2012 - 11:57:46
	ce.model.dao.RepositorioFornecedor.pesquisar(): java.sql.SQLException: Column 'codProd' not found.
joao.souza -> 03/12/2012 - 11:57:46
	ce.model.regra.ControladorFornecedor.pesquisar(): Erro ao tentar pesquisar fornecedor.
Sistema -> 03/12/2012 - 12:47:38
	ce.model.dao.RepositorioFornecedor.pesquisar(): com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: No operations allowed after statement closed.
joao.souza -> 03/12/2012 - 12:47:38
	ce.model.regra.ControladorFornecedor.pesquisar(): Erro ao tentar pesquisar fornecedor.
Sistema -> 03/12/2012 - 12:48:14
	ce.model.dao.RepositorioFornecedor.pesquisar(): com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: No operations allowed after statement closed.
joao.souza -> 03/12/2012 - 12:48:14
	ce.model.regra.ControladorFornecedor.pesquisar(): Erro ao tentar pesquisar fornecedor.
Sistema -> 03/12/2012 - 12:48:26
	ce.model.dao.RepositorioFornecedor.pesquisar(): com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: No operations allowed after statement closed.
joao.souza -> 03/12/2012 - 12:48:26
	ce.model.regra.ControladorFornecedor.pesquisar(): Erro ao tentar pesquisar fornecedor.
Sistema -> 04/12/2012 - 21:44:47
	ce.model.dao.RepositorioProduto.listar(): com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'fp.nome' in 'order clause'
joao.souza -> 04/12/2012 - 21:44:47
	ce.model.regra.ControladorProduto.listar(): Erro ao tentar listar produto.
